recipe/meta.yaml

```diff
@@ -29,6 +29,7 @@ requirements:
     - bzip2
     - xz
     - zlib
+    - lz4
```

Comment:
I think you want [`lz4-c`]( https://github.com/conda-forge/lz4-c-feedstock ). The [`lz4`]( https://github.com/conda-forge/lz4-feedstock ) package includes Python bindings to the C library.

Reply:
Should add I suspect this is causing the build issue as the `run_export` is not being used with the Python package.

Reply:
Thanks! I noted that I also forgot to add it to the run dependencies.

Reply:
I think `run_exports` should do that for you, but maybe we were getting a version of the `lz4-c` package that doesn't have `run_exports`?

Reply:
I was not aware of the `run_exports` functionality.
I can probably leave out the runtime dependencies for the other compression libraries as well then. At least if they specify `run_exports`. I will check that later.

Reply:
AFAIK it doesn't cause any problems to list them as well.

Reply:
Ok, can't hurt to be more explicit. I will leave it in then.